app-emulation/crossover-bin/crossover-bin-15.2.0.ebuild

```diff
@@ -0,0 +1,144 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
```

Comment:
I plan to keep this on EAPI=5 because I do not think I can spare the time right now to handle any regressions that the change in semantics might cause should I miss something. If you want to be responsible for any regressions caused by the change in semantics, you are welcome to switch the ebuilds to EAPI=6. :)